main.go

```diff
@@ -221,6 +230,29 @@ func main() {
 		}
 		log.Printf("The public install page url is now available in the Environment Variable: BITRISE_PUBLIC_INSTALL_PAGE_URL (value: %s)", publicInstallPage)
 	}
+
+	if len(publicInstallPages) > 0 {
+		value := strings.Join(publicInstallPages, "|")
+		if err := tools.ExportEnvironmentWithEnvman("BITRISE_PUBLIC_INSTALL_PAGE_URL_LIST", value); err != nil {
+			fail("Failed to export BITRISE_PUBLIC_INSTALL_PAGE_URL_LIST, error: %s", err)
+		}
+		log.Printf("An array of public install page urls is now available in the Environment Variable: BITRISE_PUBLIC_INSTALL_PAGE_URL_LIST (value: %s)", value)
```

Comment:
Put a newline after logging the env vars to be more readable.

Reply:
Do you mean to print a newline to the log or to put a newline in the code?